src/DiffOp/Diff/Diff.php

```diff
@@ -71,15 +73,6 @@ public function __construct( array $operations = array(), $isAssociative = null
 	}
 
 	/**
-	 * Returns the name of an interface/class that the element should implement/extend.
-	 *
-	 * @return string
-	 */
-	private function getObjectType() {
-		return '\Diff\DiffOp\DiffOp';
-	}
```

Comment:
Hah. Surprised this was still here. Been a long time since this class stopped deriving from GenericArrayObject.